SIPS/sip-7.md

```diff
@@ -198,6 +230,25 @@ interface SnapReadInterfaceRequest {
 type SnapReadInterfaceResponse = Json;
 ```
 
+#### `snap_getInterfaceState`
+
+The `snap_getInterfaceState` method lets Snaps retrive an existing user interface state. The params MUST be an object, with the following properties:
+
+- `id` (`string`): The ID of the custom UI to read.
+
+This method returns the interface state.
```

Comment:
What is "interface state"?

Reply:
The interfaces, AKA the UI has a state that stores the values of the statefull components like forms and inputs :)

Reply:
Can you clarify this in the SIP?